.github/workflows/ci.yaml

```diff
@@ -49,6 +49,7 @@ jobs:
             runalyzer-gather summary.sqlite log.sqlite
             runalyzer -m summary.sqlite -c 'dbplot(q("select $t_sim, $t_step"))'
 
+            export OMPI_MCA_rmaps_base_oversubscribe=1
```

Comment:
Could you add a comment on what this does and what problem is being addressed by this?

Reply:
Added